traffic_ops/testing/api/v4/jobs_test.go

```diff
@@ -482,6 +482,62 @@ func GetTestJobsByValidData(t *testing.T) {
 	} else {
 		t.Error("No user response available for get user by name")
 	}
+
+	// get maxRevalDurationDays parameter value
+	opts = client.NewRequestOptions()
+	opts.QueryParameters.Set("name", "maxRevalDurationDays")
+	params, _, err := TOSession.GetParameters(opts)
+	if err != nil {
+		t.Fatalf("unexpected error getting maxRevalDurationDays parameter")
+	}
+	if len(params.Response) != 1 {
+		t.Fatalf("expected 1 maxRevalDurationDays parameter, got %d", len(params.Response))
+	}
+	maxRevalDurationDays, err := strconv.Atoi(params.Response[0].Value)
+	if err != nil {
+		t.Fatalf("unexpected error converting maxRevalDurationDays value to int: %v", err)
+	}
+
+	// get jobs created within maxRevalDurationDays window
+	opts = client.NewRequestOptions()
+	opts.QueryParameters.Set("maxRevalDurationDays", "")
+	maxRevalJobs, _, err := TOSession.GetInvalidationJobs(opts)
+	if err != nil {
+		t.Errorf("unexpected error getting jobs by maxRevalDurationDays: %v", err)
+	} else if len(maxRevalJobs.Response) < 1 {
+		t.Errorf("GET /jobs by maxRevalDurationDays - expected at least 1 job")
+	}
+	for _, j := range maxRevalJobs.Response {
+		if time.Since((*j.StartTime).Time) > time.Duration(maxRevalDurationDays)*24*time.Hour {
```

Comment:
This line will segfault if the API returns a malformed response with a null or undefined `startTime`